charts/scalardb/templates/scalardb/deployment.yaml

```diff
@@ -72,15 +72,13 @@ spec:
             value: "{{ .Values.scalardb.storageConfiguration.storage }}"
           - name: SCALAR_DB_LOG_LEVEL
             value: "{{ .Values.scalardb.storageConfiguration.dbLogLevel }}"
-          - name: SCALAR_DB_SERVER_PORT
-            value: "50051"
```

Comment:
When we use default port (60051) in the Scalar DB Helm Charts, we can remove this hard-coded value from manifest file.